song-server/src/test/java/bio/overture/song/server/controller/StudyControllerTest.java

```diff
@@ -0,0 +1,33 @@
+package bio.overture.song.server.controller;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc(secure=false)
+@ActiveProfiles("dev,test")
+public class StudyControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    public void saveStudyShouldValidateStudyId() throws Exception {
+        this.mockMvc.perform(post("/studies/123/")
+                .contentType(MediaType.APPLICATION_JSON)
+                .accept(MediaType.APPLICATION_JSON)
+                .content("{\"studyId\": \"456\"}"))
+                .andExpect(status().isConflict());
+    }
+}
```

Comment:
add a new line

Reply:
I am enabling format on save, so this will not happen again